cdap-ui/app/hydrator/controllers/create/partials/nodeconfig-ctrl.js

```diff
@@ -469,6 +469,10 @@ class HydratorPlusPlusNodeConfigCtrl {
       } else {
         vm.propertyErrors = propertyErrors;
       }
+
+      if (callback && typeof callback === 'function') {
```

Comment:
@elfenheart What do we use this callback for?

Reply:
To update state in `GetSchemaWidget/index`